terraform/modules/kubernetes/variables.tf

```diff
@@ -39,6 +67,5 @@ locals {
       REDIS_URL           = local.redis_url
     }
   )
-  # Create a unique name based on the values to force recreation when they change
```

Comment:
This comment could be reworded but it's useful as the line below is quite obscure

Reply:
Comment still exists, it's on line 55, not sure why it's showing as deleted here